### PR TITLE
Update filechecker.wdl

### PR DESCRIPTION
**Pull request by aofarrel**
Now prints to an output file instead of stderr.

### CODE REVIEW

File: filechecker.wdl [hunk 1]
```diff
@@ -20,22 +20,22 @@ task stuart_filechecker {
...
-        >&2 echo "-=-=-=-=-=-=-=-=-=-"
-        >&2 echo "Contents of the truth file:"
-        >&2 cat ~{truth}
-        >&2 echo "-=-=-=-=-=-=-=-=-=-"
-        >&2 echo "cmp and diff of these files:"
-        >&2 cmp --verbose test.txt truth.txt
-        >&2 diff test.txt truth.txt
-        >&2 diff -w test.txt truth.txt
+        echo "Test checksum:" > report.txt
+        cat test.txt > report.txt > report.txt
```
Comment (DailyDreaming) [inline filechecker.wdl]: Did you mean to pipe this twice?

**Review by DailyDreaming — CHANGES_REQUESTED** [review]
I think you might want to append rather than overwriting `report.txt` each time.

See:

```
quokka@qcore 10:53 AM ~$ echo hello > test.test
quokka@qcore 10:53 AM ~$ echo hello2 > test.test
quokka@qcore 10:53 AM ~$ cat test.test 
hello2
```

**Review by DailyDreaming — CHANGES_REQUESTED** [review]
@aofarrel I think you want to use `>>` instead of `>`.

See:

```
quokka@qcore 10:53 AM ~$ echo hello >> test.test
quokka@qcore 10:57 AM ~$ echo hello2 >> test.test
quokka@qcore 10:57 AM ~$ cat test.test 
hello
hello2
```

**Comment by DailyDreaming** [comment]
@aofarrel Or use `tee`, since that will write to a file and won't swallow `stdout`.

**Comment by DailyDreaming** [comment]
@aofarrel Or use `tee`, since that will write to a file and won't swallow `stdout`.

File: filechecker.wdl (hunk 1 above)
Comment (aofarrel) [reply filechecker.wdl]: Absolutely not. Thanks for spotting this.

**Comment by aofarrel** [comment]
`tee` gives me exactly what I was looking for. Thanks!

**Pull request merged**